typos.toml

```
@@ -0,0 +1,28 @@
+[files]
+extend-exclude = [
+  "*.pbxproj",    # metadata file
+  "CHANGELOG.md", # To keep consistency between the commit history/PRs.
+]
+
+# Corrections take the form of a key/value pair. The key is the incorrect word
+# and the value is the correct word. If the key and value are the same, the
+# word is treated as always correct. If the value is an empty string, the word
+# is treated as always incorrect.
+
+# Match Whole Word - Case Sensitive
+[default.extend-identifiers]
+ser = "ser" # ron::ser - Serializer
+toi = "toi" # Time of intersention
```

Not sure what this is supposed to be, I believe you meant the time of impact?

```suggestion
toi = "toi" # Time of impact
```

Also, why is this case sensitive? I've seen TOI a lot, perhaps even more than toi.

---

Thanks it's Time of impact. I'm not familiar with the term.

about "I've seen TOI a lot, perhaps even more than toi"
I don't find TOI in the bevy repo, only toi, but I will move it to the extended words just in case.